doc/api/esm.md

```diff
@@ -801,6 +801,22 @@ property:
 
 ## Differences Between ES Modules and CommonJS
 
+### Absolute file paths don't work on Windows
+
+`import` accepts _absolute specifiers_. These are similar to _absolute paths_
+on linux and MacOS, and therefore they are also accepted.
+But on Windows, importing an _absolute path_ to a file will not work.
+To ensure cross platform compatibility, file paths should be converted into
+URLs.
+
+```js
+import { pathToFileURL } from 'url';
+
+const fileUrl = pathToFileURL('c:\\path\\to\\file.js').href;
+
+await import(fileUrl);
```

Comment:
example:

```suggestion
(async function() {
  await import(fileUrl);
})();
```

Reply:
I feel the code snipped should be short and to the point. An iife is a bit distracting. Maybe adding a ".then(...)" instead of await?

Reply:
good